src/c++/perf_analyzer/request_rate_manager.cc

```diff
@@ -196,24 +196,46 @@ RequestRateManager::PauseWorkers()
   // Pause all the threads
   execute_ = false;
 
+  ReconfigThreads();
```

Comment:
Let's pull this out (and call it ConfigureThreads, since it only does anything once).  It should be called in ChangeRequestRate() after PauseWorkers and before GenerateSchedule